packages/node_modules/@ciscospark/widget-message-meet/src/container.js

```diff
@@ -178,7 +182,14 @@ export class MessageMeetWidget extends Component {
                 image={toUserAvatarUrl}
                 name={toUser.displayName}
                 onMenuClick={this.handleMenuClick}
-              />
+              >
+                {
+                  callStartTime &&
+                  <div className={classNames(`title-bar-call-timer`, styles.callTimer)}>
```

Comment:
yea. The timer component can be used wherever and has an internal store (which I really don't like, but makes a lot of sense here). The designs specifically don't include the duration anywhere, but I do think we should have that somewhere in `widget-meet`.